bigquery_etl/cli/stage.py

```diff
@@ -387,7 +384,7 @@ def _deploy_schema(query_file):
     view_files = [
         file
         for file in artifact_files
-        if file.name == VIEW_FILE and str(file) not in dryrun.SKIP
+        if file.name == VIEW_FILE and str(file) not in dryrun.DryRun.skipped_files()
```

Comment:
should we `from ..dryrun import DryRun` and turn this into 
`if file.name == VIEW_FILE and str(file) not in DryRun.skipped_files()` like the other files?